test/evergreen.yml

```diff
@@ -1384,6 +1395,15 @@ tasks:
               ./t -1 -C "direct_io=[data]" -c ../../../test/format/CONFIG.stress
             done
 
+  - name: test-checkpoint
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ CFLAGS="-ggdb"
+          posix_configure_flags: --enable-strict
+      - func: "test checkpoint for filetype"
```

Comment:
You can move the function to here

Reply:
Ignore this.

Reply:
Fixed. Calling function with different arguments.